site/_includes/_customers.liquid

```diff
@@ -0,0 +1,27 @@
+<section class="gray">
+  <div class="container">
+    <header class="centered">
+      <h2>Trusted Worldwide</h2>
+      <p>Over 300,000 downloads on six continents.</p>
```

Comment:
The download count is hard to calculate... we don't have visibility to all of the avenues in which it can be downloaded. If we include docker it is probably more like 750k, and likely larger. 

We could just say over 500k, or .5 Million.